Python/jit.c

```diff
@@ -566,19 +555,80 @@ _PyJIT_Compile(_PyExecutorObject *executor, const _PyUOpInstruction trace[], siz
         return -1;
     }
     executor->jit_code = memory;
-    executor->jit_side_entry = memory + shim.code_size;
     executor->jit_size = total_size;
     return 0;
 }
 
+// Compiles executor in-place. Don't forget to call _PyJIT_Free later!
+static _PyJitEntryFuncPtr
+compile_trampoline(void)
+{
+    _PyExecutorObject dummy;
+    const StencilGroup *group;
+    // Loop once to find the total compiled size:
```

Comment:
Can you remove the `loop` here in the comment? It makes no sense. This is just the size of the trampoline.